tools/cldr-apps/js/test/nonbrowser/test-cldrXpathUtils.js

```diff
@@ -0,0 +1,22 @@
+const expect = require("chai").expect;
+const mocha = require("mocha");
+
+import * as cldrXpathUtils from "../../src/esm/cldrXpathUtils";
+
+describe("cldrXpathUtils test", function () {
```

Comment:
@btangmu this lets us have js tests outside of a browser (in the nonbrowser dir). I choose to use chai/mocha since you were already using it. Later we can update the browser tests to use the same chai/mocha package